tschalmers-newyorkcity.md

```diff
@@ -0,0 +1,11 @@
+### New York, NY
```

Comment:
Never been to NY, seems like the level of busyness would be overwhelming :question: